eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -0,0 +1,37 @@
+function GetPackageKey($pkg) {
```

Comment:
Do you expect to consume these helpers in the other script at a later point?

Reply:
Yes, minimize the scope of the script for now.